sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/simple/AggregateTests.java

```diff
@@ -88,6 +89,67 @@ public void testIssue1957() {
                 (2 rows)""");
     }
 
+    @Test
+    public void nullTumble() {
+        this.compileRustTestCase("""
+                create table PRICE (ts TIMESTAMP);
+                create view x AS
+                SELECT *
+                FROM TABLE(
+                  TUMBLE(
+                    DATA => TABLE price,
+                    TIMECOL => DESCRIPTOR(ts),
+                    SIZE => INTERVAL '1' HOUR));""");
+    }
+
+    @Test
+    public void aggTimestamp() {
+        this.compileRustTestCase("""
```

Comment:
It shouldn't be hard to add an input/output test here?

Reply:
almost nothing I do is that hard, but everything takes time.
this wasn't compiling, but now it is.
I am not sure what other database I can use to test this program, so I will have to compute the output by hand.

Reply:
I have added some test data. Seems to work fine.